src/views/ActionPlan/index.tsx

```diff
@@ -0,0 +1,448 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { createUseStyles } from "react-jss";
+import { CurrentUser } from "../../types";
+import { Form, Formik } from "formik";
+import * as Yup from "yup";
+import { useSnackbar } from "notistack";
+import {
+  Button,
+  FormLabel,
+  Textarea,
+  Slider,
+  SliderFilledTrack,
+  SliderMark,
+  SliderThumb,
+  SliderTrack,
+  Stack,
+  Input,
+  FormControl,
+} from "@chakra-ui/react";
+import axios from "axios";
+import envConfig from "../../envConfig";
+
+const useStyles = createUseStyles({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    textAlign: "flex-start",
+    "& > div": {
+      padding: [10, 0],
+    },
+  },
+  heading: {
+    fontSize: 24,
+    margin: [20, 0],
+  },
+  sliders: {
+    "& > label": {
+      paddingBottom: 15,
+    },
+  },
+  errors: {
+    color: "red",
+    padding: [5, 0],
+  },
+});
+
+type ActionPlanProps = {
+  currentUser: CurrentUser;
+};
+
+const ActionPlan = ({ currentUser }: ActionPlanProps): JSX.Element => {
+  const classes = useStyles();
+  const { enqueueSnackbar } = useSnackbar();
+  const navigate = useNavigate();
+
+  const handleSubmit = async (values: any) => {
+    const keyActionItems = [
+      values.keyActionItem1,
+      values.keyActionItem2,
+      values.keyActionItem3,
+      values.keyActionItem4,
+      values.keyActionItem5,
+    ];
```

Comment:
can we just spread these values? `[...values]`

Reply:
It's not all the values, just the 5 key action times